example/main.tf

```diff
@@ -85,6 +85,38 @@ resource "google_compute_subnetwork" "vpc_subnetwork" {
   ]
 }
 
+resource "google_compute_router" "router" {
+  name    = "${var.router_name}"
+  network = "${google_compute_network.vpc_network.self_link}"
+
+  bgp {
+    asn = 64514
+  }
+}
+
+resource "google_compute_address" "address" {
+  count = 1
```

Comment:
So this give the NAT gateway a single IP address? Might be worth making this configurable if it's easy to do 😄

Reply:
I'm not really sure what I was doing here but it's gone now as it's not required.